zebracrossing/campaigns/urls.py

```diff
@@ -5,6 +5,7 @@
 app_name = "campaigns"
 urlpatterns = [
     path("", views.index, name="index"),
+    path("save_to_table", views.save_to_table, name="save_to_table"),
```

Comment:
This path should be `<int:pk>/schedule/save`, where `<int:pk>` is the place holder for the campaign ID

Also the name should be `save_schedule`